### ISSUE TITLE
Ability to use Zeus with invoice(s) macaroon

### ISSUE THREAD

**Issue by yanascz** [ZeusLN/zeus#1300]
### Describe the feature

As a user, I'd like to use Zeus with invoice(s) or other non-admin macaroons. The app could only show the tabs I'm able to use. The invoice macaroon seems to be the bare minimum requirement.

The Bitcoin tab could be used as the default screen, same as now. As long as I'm allowed to see the balance, the Home tab might be available. As long as I can view existing channels, the Channels tab would be present. Ability to open channels would be available only if authorized to do so.

I assume there are other approaches to this, the above are just examples. Any solution with restricted macaroons would be appreciated.

**Comment by kaloudis** [comment]
I want to see this too. We have a clearpath forward to at least support this with LNC. Other APIs may prove to be a bit more difficult, but I think we can find a way.

**Comment by yanascz** [comment]
Thinking about it a bit more, there might be alternative/universal solutions. Let's say I run a café and I want my staff to be able to receive LN payments but I don't want them to see my node's balance or change any settings. What about allowing me to configure the app to just show relevant tabs and set an admin password?

Using a non-admin macaroon seems to be an ideal solution but an in-app configuration might be a good first step. Or maybe a combination of both approaches? 🤔

**Comment by kaloudis** [comment]
If you're running the new POS, you can add a PIN or Password to the app and they'll be able to process orders but not see the balance -- that being said not everyone is using that mode as it is Square only for now.

**Comment by MateoGuinoe** [comment]
Yes, an employee mode would be great. Watch only wallet that would have unified QRs (BIP21?) and the node total sats hidden yet employees could see recent history to verify a payment was made correctly (like when end of day till count is off - etc).

**Comment by brizbomb** [comment]
Yes, an employee mode would be great. Watch only wallet that would have unified QRs (BIP21?) and the node total sats hidden yet employees could see recent history to verify a payment was made correctly (like when end of day till count is off - etc).
